WordPress/wporg-main-2022
# Patterns: Update the WP20 banner, use Link Wrapper block

This PR switches the large banner on the homepage to a smaller banner, wrapped in a link. This also updates the content of the banner to highlight the WP20 website, instead of the swag.

Fixes #258, See https://github.com/WordPress/wporg-mu-plugins/pull/398 (required before this can be merged)

cc @DanSoschin

### Screenshots

| Size | Screenshot |
|--------|-------|
| 400px | <img alt="" width="400" src="https://github.com/WordPress/wporg-main-2022/assets/541093/f88d4b9e-670f-47e4-8d86-d578f67a480c" /> |
| 600px | <img alt="" width="600" src="https://github.com/WordPress/wporg-main-2022/assets/541093/704ca502-c6ff-4cad-bc1d-3d66cdf6b49c" /> |
| 960px | ![960](https://github.com/WordPress/wporg-main-2022/assets/541093/29774fd5-f0ff-4611-9f7f-67c6e6e63f02) |
| 1400px | ![1400](https://github.com/WordPress/wporg-main-2022/assets/541093/dc061e3f-060d-4416-b7fd-995fadd16c85) |
| Focus state | ![focus](https://github.com/WordPress/wporg-main-2022/assets/541093/3d1ad33b-4fdb-4abf-b0c7-77604a454135) |
| Hover state | ![hover](https://github.com/WordPress/wporg-main-2022/assets/541093/7c6b0faf-fb47-4922-98f9-ad396378cf7f) |

### How to test the changes in this Pull Request:

1. Make sure https://github.com/WordPress/wporg-mu-plugins/pull/398 is applied or already merged
2. Check out this PR, build the CSS
3. View the homepage, it should have the smaller banner.
